monitor/monitor_cyclic_reboot.go

```diff
@@ -0,0 +1,22 @@
+package monitor
+
+import (
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/blockassets/bam_agent/service"
+)
+
+func monitorCyclicReboot(delayInHours int) {
+	s1 := rand.NewSource(time.Now().UnixNano())
+	r1 := rand.New(s1)
+	// We dont want to reset miners too close together... if multiple miners are reset or added at same time,
+	// this ensures there is a spread on the time to restart the miner
+	timeToWait := time.Duration(delayInHours)*time.Hour + time.Duration(r1.Intn(3600))*time.Second
```

Comment:
Add constant for 3600

Reply:
Since this is in two places now, maybe it should be a function somewhere else that builds the timeToWait?